webpack/webpack-workspace-resolve-plugin/src/WorkspaceResolvePlugin.ts

```diff
@@ -26,48 +32,53 @@ export interface IWorkspaceResolvePluginOptions {
  */
 export class WorkspaceResolvePlugin implements WebpackPluginInstance {
   private readonly _cache: WorkspaceLayoutCache;
+  private readonly _resolverNames: Set<string>;
 
-  public constructor(cache: WorkspaceLayoutCache) {
-    this._cache = cache;
+  public constructor(options: IWorkspaceResolvePluginOptions) {
+    this._cache = options.cache;
+    this._resolverNames = new Set(options.resolverNames ?? ['normal']);
```

Comment:
Should we default to this? Or default to resolving everything?

Reply:
Switched to all of "normal", "context", "loader"; there are other resolver names, but those are the most common ones.